src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -17,6 +17,9 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.Extensions.Configuration;
 using Microsoft.TeamFoundation.Common;
+using MongoDB.Driver;
```

Comment:
Why MongoDb Driver?

Reply:
This is also not something I added, at least not intentionally. Tested by commenting it out, still works. Will delete for now.